reference/api/stats.md

```diff
@@ -34,6 +48,8 @@ Get stats of an index.
 }
 ```
 
+ In the above code sample, `fieldDistribution` shows every field in the `movies` index along with the total number of documents in `movies` that contain that field.
```

Comment:
I don't think either of these (the two pieces of content that are almost identical) are necessary if we have a full explanation of the returned fields above. We can just link to that instead.